core/src/banking_stage/immutable_deserialized_packet.rs

```diff
@@ -42,33 +39,45 @@ pub struct ImmutableDeserializedPacket {
     transaction: SanitizedVersionedTransaction,
     message_hash: Hash,
     is_simple_vote: bool,
-    priority_details: TransactionPriorityDetails,
+    priority: u64,
+    compute_unit_limit: u64,
 }
 
 impl ImmutableDeserializedPacket {
     pub fn new(packet: Packet) -> Result<Self, DeserializedPacketError> {
         let versioned_transaction: VersionedTransaction = packet.deserialize_slice(..)?;
-        let sanitized_transaction = SanitizedVersionedTransaction::try_from(versioned_transaction)?;
+        let is_simple_vote = packet.meta().is_simple_vote_tx();
+        // drop transaction if prioritization fails.
+        //
+        // TODO - wire up feature_set from recent bank. For now, mimic current implementation of
```

Comment:
potential issues around feature activations. If we use a recent (rooted?) bank, we likely won't be able to **actually** use features until several slots after the epoch boundary.

Reply:
current code uses hard coded feature status for immutable_deserialzied_packet, which this code mimics. Perhaps this is a better/stable option compare to use highest_slot from bank_fork to get feature_set.